redhat-cop/uncontained.io
# Adding new homepage layout; consolidating guides and articles

#### What is this PR About?
This change introduces a tiled homepage layout, based on wireframes from @lwrigh.

#### How should we test or review this PR?
```
npm install
npm start
```

- Check homepage, make sure everything looks ok and is well aligned. 
- Click into "View all content", see that everything 
- Check that articles still render properly.

#### Is there a relevant Trello card or Github issue open for this?

#55 
#26 
#57 

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
